Documentation/help_files/report_dashboard_help.php

```diff
@@ -0,0 +1,334 @@
+                    
+                        <p><?php echo xlt("The basis for CCR is a Patient Care Referral Form developed by the Massachusetts Department of Public Health");?>.
+                        
+                        <p><?php echo xlt("In total, there are seventeen sections that can be included within a Continuity of Care Record (CCR)");?>.
+                        
+                        <p id = "my_hide"><?php echo xlt("All sections are not required, rather, they are available based on the information a practitioner believes is critical data for the patient at any given moment in time");?> &nbsp <i class="show_hide fa fa-eye fa-lg small" title="<?php echo xla('Click to Show'); ?>"></i>
+                        
+                            <div class='hideaway' id='ccr_sections'style='display: none;'>
+                                <ul>
+                                    <li><?php echo xlt("Patient Demographics"); ?></li>
+                                    <li><?php echo xlt("Immunizations"); ?></li>
+                                    <li><?php echo xlt("Vital Signs"); ?></li>
+                                    <li><?php echo xlt("Problems & Diagnoses"); ?></li>
+                                    <li><?php echo xlt("Insurance Information"); ?></li>
+                                    <li><?php echo xlt("Health Care Providers"); ?></li>
+                                    <li><?php echo xlt("Encounter Information"); ?></li>
+                                    <li><?php echo xlt("Allergies/Alerting Data"); ?></li>
+                                    <li><?php echo xlt("Appropriate Results"); ?></li>
+                                    <li><?php echo xlt("Medication"); ?></li>
+                                    <li><?php echo xlt("Procedures"); ?></li>
+                                    <li><?php echo xlt("Results"); ?></li>
+                                    <li><?php echo xlt("Necessary Medical Equipment"); ?></li>
+                                    <li><?php echo xlt("Social History"); ?></li>
+                                    <li><?php echo xlt("Statistics"); ?></li>
+                                    <li><?php echo xlt("Family History"); ?></li>
+                                    <li><?php echo xlt("Care Plan"); ?></li>
+                                </ul>
+                            </div>
+                        <div>
+                        <p><?php echo xlt("The core dataset consists of");?>: &nbsp;<i class="show_hide fa fa-eye fa-lg small" title="<?php echo xla('Click to Show'); ?>"></i>
+                        
+                        <div class='hideaway' id='ccr_core'style='display: none;'>
+                    
+                            <ul>
+                                <li><?php echo xlt(' Document identifying information');?>*</li>
```

Comment:
remove the leading space in this constant

Reply:
Done